android/mfx_defs.mk

```diff
@@ -117,6 +117,8 @@ MFX_INCLUDES := $(LOCAL_PATH)/include
 
 MFX_INCLUDES_LIBVA := $(TARGET_OUT_HEADERS)/libva
```

Comment:
There should be no such folder $(TARGET_OUT_HEADERS)/libva if LOCAL_COPY_HEADERS_TO is removed in libva Android.mk...

Thanks,
Tianmi

Reply:
sure, thanks!